CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Code of conduct
+
+* We are committed to providing a friendly, safe and welcoming environment for
+all, regardless of level of experience, gender identity and expression, sexual
+orientation, disability, personal appearance, body size, race, ethnicity, age,
+religion, nationality, or other similar characteristics.
+* Please avoid using overtly sexual aliases or other nicknames that might
+detract from a friendly, safe and welcoming environment for all.
+* Please be kind and courteous. There’s no need to be mean or rude.
+* Respect that people have differences of opinion and that every design or
+implementation choice carries a trade-off and numerous costs. There is seldom
+a right answer.
+* Please keep unstructured critique to a minimum. If you have solid ideas you
+want to experiment with, make a fork and see how it works.
+* We will exclude you from interaction if you insult, demean or harass anyone.
+That is not welcome behavior. We interpret the term “harassment” as including
+the definition in the
+[Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md);
+if you have any lack of clarity about what might be included in that concept,
+please read their definition. In particular, we don’t tolerate behavior that
+excludes people in socially marginalised groups.
+* Private harassment is also unacceptable. No matter who you are, if you feel
+you have been or are being harassed or made uncomfortable by a community member,
+please contact one of the channel ops or any of the
+[shuttle moderators](mailto:mods@shuttle.rs) immediately. Whether you’re a
```

Comment:
I don't think we have mods@shuttle.rs - consider changing to opensource@shuttle.rs